postreise/plot/plot_scatter_capacity_vs_capacity_factor.py

```diff
@@ -0,0 +1,96 @@
+import matplotlib.pyplot as plt
+import pandas as pd
+
+from postreise.analyze.generation.capacity import (
+    get_capacity_by_resources,
+    get_capacity_factor_time_series,
+)
+from postreise.analyze.helpers import get_plant_id_for_resources_in_area
+from postreise.analyze.time import change_time_zone, slice_time_series
+
+
+def plot_scatter_capacity_vs_capacity_factor(
+    scenario,
+    area,
+    resources,
+    time_zone="utc",
+    time_range=None,
+    area_type=None,
+    between_time=None,
+    dayofweek=None,
+    markersize=50,
+    fontsize=20,
+    title=None,
+    show_plot=True,
+):
+    """Generate scatter plot with x-axis being capacity and y-axis being capacity
+    factor for a scenario in the specific area during any interested time range.
```

Comment:
Generate for a given scenario the scatter plot of the capacity (x-axis) vs capacity factor (y-axis) of generators located in area and fueled by resources over a time range.

Reply:
Done.